Model/DutyCalculator.php

```diff
@@ -655,11 +650,11 @@ protected function getQuote($request, $accessToken)
         // $this->_logger->debug('----------------GET QUOTE FROM DHL - START OF REQUEST----------------');
         // $this->_logger->debug(json_encode($url));
         // $this->_logger->debug(json_encode($accessToken));
-        // $this->_logger->debug(json_encode($request));
+        $this->_logger->debug(json_encode($request));
         // $this->_logger->debug('================GET QUOTE FROM DHL - END OF REQUEST================');
```

Comment:
is it worthwhile to delete these other commented debug statements? I see the value in the debug statements that you've left in uncommented.

Reply:
About the commented out debug level log statements - those are from Ryan's and I can definitely remove. I un-commented the one that I used. The uncommented one would not cause any problem on production as there is a magento command  to turn this on or off. On production it would be off. We had a lengthy discussion on this back in January. Recommended approach is leaving logging statements as is (instead of commenting those out) and turning on and off those based on which env it is - dev/qa/prod. This is true and supported in most of the programming languages.

Reply:
awesome, if you get to it before the review is closed then great! If not, that's ok. It's a small thing.